source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -0,0 +1,465 @@
+=================
+Server Monitoring
+=================
+
+:Spec: 1580
+:Title: Server Monitoring
+:Author: Shane Harvey
+:Advisors: Matt broadstone
+:Status: Accepted
+:Type: Standards
+:Version: 2.15
+:Last Modified: 2020-02-20
+
+.. contents::
+
+--------
+
+Abstract
+--------
+
+This spec defines how a driver monitors a MongoDB server. In summary, the
+client monitors each server in the topology. The scope of server monitoring is
+to provide the topology with updated ServerDescriptions based on isMaster
+command responses.
+
+META
+----
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in `RFC 2119 <https://www.ietf.org/rfc/rfc2119.txt>`_.
+
+Specification
+-------------
+
+Terms
+'''''
+
+See the terms in the `main SDAM spec`_.
+
+.. _checking: #check
+.. _checks: #checks
+
+check
+`````
+
+The client checks a server by attempting to call ismaster on it,
+and recording the outcome.
+
+.. _scans: #scans
+
+scan
+````
+
+The process of checking all servers in the deployment.
+
+suitable
+````````
+
+A server is judged "suitable" for an operation if the client can use it
+for a particular operation.
+For example, a write requires a standalone
+(or the master of a master-slave set),
+primary, or mongos.
+Suitability is fully specified in the `Server Selection Spec`_.
+
+Monitoring
+''''''''''
+
+The client monitors servers by `checking`_ them periodically,
+pausing heartbeatFrequencyMS between checks.
+Clients check servers sooner in response to certain events.
+
+The socket used to check a server MUST use the same
+`connectTimeoutMS <http://docs.mongodb.org/manual/reference/connection-string/>`_
+as regular sockets.
+Multi-threaded clients SHOULD set monitoring sockets' socketTimeoutMS to the
+connectTimeoutMS.
+(See `socket timeout for monitoring is connectTimeoutMS`_.
+Drivers MAY let users configure the timeouts for monitoring sockets
+separately if necessary to preserve backwards compatibility.)
+
+The client begins monitoring a server when:
+
+* ... the client is initialized and begins monitoring each seed.
+  See `initial servers`_.
+* ... `updateRSWithoutPrimary`_ or `updateRSFromPrimary`_
+  discovers new replica set members.
+
+When checking a server, clients MUST NOT include SCRAM mechanism
```

Comment:
Yeah this line also stood out to me. Even if it's not redundant I think it could be moved to a better place, like the section that says not to authenticate.